pkg/reconciler/names.go

```diff
@@ -21,30 +21,26 @@ import (
 
 	"strings"
 
+	"github.com/knative/serving/pkg/reconciler/v1alpha1/autoscaling/kpa/resources/names"
 	"github.com/knative/serving/pkg/utils"
 )
 
-const (
-	serviceSuffix = "-service"
-	metrixSuffix  = "-metrics"
-)
-
 // GetK8sServiceFullname returns service full name
 func GetK8sServiceFullname(name string, namespace string) string {
 	return fmt.Sprintf("%s.%s.svc.%s", name, namespace, utils.GetClusterDomainName())
 }
 
 // GetMetricsK8SServiceNameForObj returns the metrics name for the object
 func GetMetricsK8SServiceNameForObj(name string) string {
-	return name + metrixSuffix
+	return names.MetricsServiceName(name)
```

Comment:
Why I meant by my prior comment was that I don't understand why we need these defined here at all.  Why not have the callers use the naming function directly?

Reply:
I think some time ago, someone added those as a universal place., hiding implementation details.
I am fine either way.

Reply:
Yeah, there used to be a lot more here and I got rid of almost all in favor of `./pkg/reconciler/vX/bar/resources/names`.  If we can avoid adding this here, then let's avoid it.

Reply:
Done. I did not remove the others. I'll clean that up separately, this change is big as is.